pkg/mimirpb/compat_test.go

```diff
@@ -483,6 +530,16 @@ func TestFromFloatHistogramToHistogramProto(t *testing.T) {
 	assert.Equal(t, expected, p2)
 }
 
+func BenchmarkFromFloatHistogramToHistogramProto(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		for i := 0; i < 100; i++ {
```

Comment:
Same as above: is it possible to move the setup work outside the benchmark loop?